ProStudio/ProStudio/Classes/Screens/ProjectDiscussion/ProjectDiscussion.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  ProjectDiscussionTableViewController.swift
+//  ProStudio
+//
+//  Created by Zimma on 10/10/2018.
+//  Copyright © 2018 Nikita. All rights reserved.
+//
+
+import UIKit
+
+class ProjectDiscussion: UITableViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView.separatorColor = .clear
+        tableView.isScrollEnabled = false
+        tableView.allowsSelection = false
+        
+    }
+    
+    // MARK: - Table view data source
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 6
+    }
+
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+
+        if indexPath.row == 0 {
+            
+            let cell  = Bundle.main.loadNibNamed("TextFieldCell", owner: self, options: nil)?.first as! TextFieldCell
```

Comment:
invalid cell initializing.

You need to register cells first. Ask how it doing in #freshmen chat. And after deque them with tableView